Recipes/Recipes/ProfileModule/View/BonusesView.swift

```diff
@@ -0,0 +1,146 @@
+// BonusesView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран профиля пользователя
+final class BonusesView: UIViewController {
+    
+    // MARK: - Constants
+    
+    enum Constants {
+        static let dieImage = UIImage(named: "die")
+        static let titleText = "Your bonuses"
+        static let verdanaBold20 = UIFont(name: "Verdana-Bold", size: 20)
+        static let closeButtonImage = UIImage(named: "close")
+        static let verdana14 = UIFont(name: "Verdana", size: 14)
+        static let presentImage = UIImage(named: "present")
+        static let bigStarImage = UIImage(named: "bigStar")
+        static let oneHundredImage = UIImage(named: "100")
+    }
+    
+    // MARK: - Visual Components
+    
+    private let dieImageView: UIImageView = {
```

Comment:
Везде аннотацию типа надо убрать, он из контекста поймет
```suggestion
    private let dieImageView = {
```